lib/templates/android-kotlin/entity.liquid

```diff
@@ -0,0 +1,50 @@
+{%- if params.package.size > 0 -%}
+package {{ params.package }}
+{%- endif %}
+
+/* DO NOT EDIT | Generated by gyro */
+{{ empty_line }}
+
+{%- comment %} ******* IMPORTS ******* {% endcomment -%}
+{%- if entity.has_json_key_path == true %}
+import com.google.gson.annotations.SerializedName
+{% comment %} *** Empty line *** {% endcomment %}
+{%- endif %}
+{%- if entity.has_date_attribute == true %}
+import java.util.Date
+{%- endif %}
+{%- if entity.has_list_relationship == true %}
+import java.util.List
+{%- endif %}
+{%- if entity.has_date_attribute == true or entity.has_list_relationship == true %}
+{{ empty_line }}
+{%- endif %}
+{%- if entity.has_list_attributes == true %}
+import io.realm.RealmList
+{%- endif %}
+import io.realm.RealmObject
+{%- if entity.has_ignored == true %}
+import io.realm.annotations.Ignore
+{%- endif %}
+{%- if entity.has_indexed_attributes == true %}
+import io.realm.annotations.Index
+{%- endif %}
+{%- if entity.has_primary_key == true %}
+import io.realm.annotations.PrimaryKey
+{%- endif %}
+import io.realm.annotations.Required
```

Comment:
For consistency, we should import that conditionally, the same way that other annotations are imported conditionally, so that it's not imported if no property is required.

Reply:
I'm all for importing that conditionally  but I'm not sure if there is a `has_non_optionnal_fiels` property on the entity object ?

Reply:
C'est bon en fait j'ai trouvé `has_required`